src/renderer/inputSystem/InputSystem.ts

```diff
@@ -93,8 +95,10 @@ export class InputSystem {
 
   private update = () => {
     if (this.isRunning) {
-      this.updateGamepad();
-      requestAnimationFrame(this.update);
+      setTimeout(() => {
+        this.updateGamepad();
+        requestAnimationFrame(this.update);
+      }, 1000 / GAMEPAD_REFRESH_RATE);
```

Comment:
The `` 1000 `` is to convert from a unit to another I presume. Might want to make that number a constant too.

Reply:
The 1000 is just because setTimeout takes in the delay in ms and not seconds so we need to make a conversion because Hz are per second and not millisecond.

Reply:
In my opinion moving that into a constant will just make the code more confusion but that's just my opinion

Reply:
I have other ideas:
Make the ``1000 / GAMEPAD_REFRESH_RATE`` itself a constant.
Put a comment on line 101.
etc.

If you don't agree, this doesn't matter much frankly.

Reply:
Yeah sure good idea